src/app.ts

```diff
@@ -197,13 +199,15 @@ async function app() {
   console.log();
   printResults(outPackage);
 
+  console.log();
+  await checkSushiVersion();
+
   process.exit(stats.numError);
 }
 
 function getVersion(): string {
-  const packageJSONPath = path.join(__dirname, '..', 'package.json');
-  if (fs.existsSync(packageJSONPath)) {
-    const sushiVersion = fs.readJSONSync(packageJSONPath)?.version;
+  const sushiVersion = getLocalSushiVersion();
+  if (sushiVersion !== 'unknown') {
```

Comment:
I think this should now be checking for `null` rather than `'undefined'`.

Reply:
Err, rather than `'unknown'`. Sorry about that.

Reply:
Done, should I also change the 'unknown' return in function getVersion() in app.ts or should I just keep it as-is?